framework/doc/content/source/postprocessors/PseudoTimestep.md

```diff
@@ -0,0 +1,46 @@
+# PseudoTimestep
+
+The methods implemented in this postprocessor compute timesteps adaptively using Pseudo Transient Continuation strategies. This allows for steady state problems solved via time marching approcah to reach the final state rapidly.
```

Comment:
```suggestion
The methods implemented in this postprocessor compute timesteps adaptively using Pseudo Transient Continuation strategies. This allows for steady state problems solved via time marching approach to reach the final state rapidly.
```

Reply:
The .md documentation is to be extended, and I'd of course run a spell checker at the end, but as a draft, it felt overkill.